src/Chae/Tree.elm

```diff
@@ -182,7 +182,7 @@ First argument is `Maybe Id` is ether:
 - `Nothing` => result is given tree (with empty ancestors `List`).
 - `Just parentId` => result is sub tree for node with `id == parentId`.
 
-Returns tuple containing sub tree and list of ancestors of `Node`.
+Returns tuple containing sub tree and list of ancestors (paratenrs of root `Node`).
```

Comment:
Sorry, rebasing. I removed the parentheses with the typo in #2 because it didn't make much sense to me ("ancestor" is a standard term, using "root" to describe the type `Node` wraps in this context is confusing).

Thinking about it, the function returns the ancestors of the sub tree (ancestors of the `Node` identified by the `id` argument + that `Node`). Therefore, I'd change it to "... and list of ancestors of that sub tree."

Plus the list goes from the sub tree to root, not the other way around. It is apparent, looking at the examples, but it might be worth mentioning explicitly.

Which version do you prefer?